buildpipeline/DotNet-CoreFx-Trusted-Windows.json

```diff
@@ -424,6 +424,9 @@
     },
     "PB_CleanAgent": {
       "value": "true"
+    },
+    "PB_SymbolsBuildId": {
+      "value": "DotNet-CoreFx-Windows"
```

Comment:
Actually, I don't think we need "Windows", I just replaced it with System.DefinitionId